plugins/kubesec

```diff
@@ -0,0 +1 @@
+repository = https://github.com/vitalis/asdf-kubesec
```

Comment:
This should be the same Git URL shown in the project's readme. `https://github.com/vitalis/asdf-kubesec.git`

Reply:
@Stratus3D fixed